src/utils/isIndexedDBValid.js

```diff
@@ -1,36 +1,67 @@
 import idb from './idb';
 
+function getSafariUserAgentVersion(userAgent) {
+    try {
+        // https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent
+        const match = /Version\/([0-9._]+)/.exec(userAgent);
+        let versionParts = match && match[1].split(/[._]/).slice(0, 2);
+        if (!versionParts || !versionParts.length) {
+            return;
+        }
+
+        const version = Number(versionParts.join('.'));
+        return version;
+    } catch (e) {
+        return;
+    }
+}
+
+// export for testing, will not end up exported in the dist files
```

Comment:
I just wanted to clarify why I'm exporting this, even though no other module imports it.
So, this export is here so that we can import in in the unit tests but it will not be reexported in the dist and will be minified as usual.